tests/sweep_framework/sweeps/eltwise/binary/add/add_set2_pytorch2.py

```diff
@@ -0,0 +1,667 @@
+            {"self": [1, 96, 14, 14], "other": [1, 96, 14, 14]},
+            {"self": [1, 96, 19, 19], "other": [1, 96, 19, 19]},
+            {"self": [1, 96, 56, 56], "other": [1, 96, 56, 56]},
+            {"self": [1, 96, 7, 7], "other": [1, 96, 7, 7]},
+            {"self": [1, 960, 7, 7], "other": [1, 960, 7, 7]},
+            {"self": [1, 98, 28, 28], "other": [1, 98, 28, 28]},
+            {"self": [10, 10], "other": [10, 10]},
+            {"self": [100, 1, 256], "other": [100, 1, 256]},
+            {"self": [12, 24, 24], "other": [12, 24, 24]},
+            {"self": [13600, 1, 4], "other": [1, 9, 4]},
+            {"self": [15, 15], "other": [15, 15]},
+            {"self": [16, 6, 49, 49], "other": [1, 6, 49, 49]},
+            {"self": [16, 6, 64, 64], "other": [1, 6, 64, 64]},
+            {"self": [16, 8, 49, 49], "other": [1, 8, 49, 49]},
+            {"self": [16, 8, 64, 64], "other": [1, 8, 64, 64]},
+            {"self": [2, 512], "other": [2, 512]},
+            {"self": [2, 7, 2048], "other": [2, 7, 2048]},
+            {"self": [2, 7, 512], "other": [1, 7, 512]},
+            {"self": [2, 7, 512], "other": [2, 7, 512]},
+            {"self": [2, 8, 7, 7], "other": [2, 1, 7, 7]},
+            {"self": [2048, 262], "other": [262]},
+            {"self": [221, 1, 4], "other": [1, 9, 4]},
+            {"self": [25, 4], "other": [25, 1]},
+            {"self": [3234, 1], "other": [3234, 1]},
+            {"self": [3234, 2], "other": [3234, 2]},
+            {"self": [3234], "other": [3234]},
+            {"self": [3400, 1, 4], "other": [1, 9, 4]},
+            {"self": [4, 12, 49, 49], "other": [1, 12, 49, 49]},
+            {"self": [4, 12, 64, 64], "other": [1, 12, 64, 64]},
+            {"self": [4, 16, 49, 49], "other": [1, 16, 49, 49]},
+            {"self": [4, 16, 64, 64], "other": [1, 16, 64, 64]},
+            {"self": [59, 1024], "other": [59, 1024]},
+            {"self": [63, 1, 4], "other": [1, 9, 4]},
+            {"self": [64, 3, 49, 49], "other": [1, 3, 49, 49]},
+            {"self": [64, 3, 64, 64], "other": [1, 3, 64, 64]},
+            {"self": [64, 4, 49, 49], "other": [1, 4, 49, 49]},
+            {"self": [64, 4, 64, 64], "other": [1, 4, 64, 64]},
+            {"self": [850, 1, 4], "other": [1, 9, 4]},
+            {"self": [8732, 1], "other": [8732, 1]},
+            {"self": [8732, 2], "other": [8732, 2]},
+            {"self": [8732], "other": [8732]},
+            {"self": [920, 1, 256], "other": [256]},
+            {"self": [920, 1, 256], "other": [920, 1, 256]},
+        ],
+        "input_a_dtype": [ttnn.bfloat16],
+        "input_b_dtype": [ttnn.bfloat16],
+        "input_a_layout": [ttnn.TILE_LAYOUT],
+        "input_b_layout": [ttnn.TILE_LAYOUT],
+        "input_a_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
+        "input_b_memory_config": [ttnn.DRAM_MEMORY_CONFIG, ttnn.L1_MEMORY_CONFIG],
+    },
+    # suite for scalar/0dim
+    "nightly": {
```

Comment:
verified whether both cases are picking up? are both are same suite name?

Reply:
Oh. as this was tested on local, they were different suite names. 
In this scenario should "nightly" the suite name for both suites or different.. ? @Aswinmcw

Reply:
This can be verified locally by renaming this to new file then running the sweep with nightly suite name, that will show the total cases picked up